stable/metricbeat/templates/serviceaccount.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.deployment.enabled }}
```

Comment:
Just a quick look, maybe I'm mistaken, but shouldn't these be `if (or .Values.deployment.enabled .Values.daemonset.enabled))`?

Otherwise I suspect daemonset pods will end up missing RBAC resources even in case they're demanded.

Reply:
This seems to be my mistake. I removed this condition.

Thanks for pointing this out :)